lib/json_api_helpers/lib/json_api_helpers/version.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+module JsonApiHelpers
+  MAJOR = '0'.freeze
+  MINOR = '1'.freeze
+  PATCH = '0'.freeze
```

Comment:
These constants really don't have to be freezed since this file already has the "magic comment" that automatically freezes all strings